.changeset/red-apricots-teach.md

```diff
@@ -0,0 +1,6 @@
+---
+"@khanacademy/perseus": minor
+"@khanacademy/perseus-editor": minor
```

Comment:
Making this `minor` and not `patch` because `perseus-api.jsx` has a warning about bumping the version.

Reply:
That warning isn't about the version of the npm packages we publish.  It's about the version in version.js.  If this is a breaking change (which I think it is), we should be bumping these packages (or at the very least `@khanacademy/perseus`) by a major version number.  The current API version number in version.js is actually `11.0`.

Reply:
I'm a little confused why we have semver and also an API version in `version.js`; I thought semver was for API versioning. Either way, I made the changeset major and bumped `version.js`, lmk if I'm misunderstanding the ask.

Reply:
`version.js` pre-dates our use of semver.  For the longest time we never published `perseus` as an npm module so we didn't really care.  Honestly, we could probably just drop `apiVersion` from version.js.  I don't see it being used anywhere in our repos. @jeremywiebe what do you think?

Reply:
I'd agree with removing `apiVersion` (we do need to keep `itemDataVersion` as that is used by widgets and Perseus itself to version the data it deals with. 

I do see a file in webapp named `services/static/javascript/perseus-one-package/calculate-perseus-version.js` which deals with `apiVersion`. That file is only ever imported by `services/static/javascript/perseus-one-package/item-area.jsx`. I can't quite work out what it's doing, but it'd be good to verify removing the `apiVersion` won't cause issues there before doing that.

Reply:
Good catch.  I was grepping for `apiVersion` and didn't get any hits.

Reply:
That file looks like it's setting `perseusApiMajorVersion` on some object.  I grepped for `perseusApiMajorVersion` and got a lot of hits.  It looks like it's being used in a few queries so it may actually be important.

Reply:
After looking at the code a bit more, it looks like we use `perseusApiMajorVersion` in the editor to see this property on assessment item data in the data store.  We query for this property in several queries, but I can't see anything that actually checks the value of this property.